src/Appwrite/Platform/Workers/Deletes.php

```diff
@@ -405,7 +405,7 @@ private function deleteCacheByDate(Document $project, callable $getProjectDB, st
         );
 
         $queries = [
-            Query::select($this->selects),
+            Query::select(array_merge($this->selects, ['accessedAt'])),
```

Comment:
Let's use the `['accessedAt', ...$this->selects]` pattern instead of array_merge

Reply:
Done!